libcloud/test/dns/test_auroradns.py

```diff
@@ -47,17 +54,180 @@ def test_merge_extra_data(self):
                 param: params[param]
             }
 
-            data = self.driver._AuroraDNSDriver__merge_extra_data(rdata,
-                                                                  extra)
+            data = self.driver._AuroraDNSDriver__merge_extra_data(rdata, extra)
             self.assertEqual(data['content'], '127.0.0.1')
             self.assertEqual(data['type'], RecordType.A)
             self.assertEqual(data[param], params[param])
             self.assertEqual(data['name'], 'localhost')
 
+    def test_list_zones(self):
+        zones = self.driver.list_zones()
+        self.assertEqual(len(zones), 2)
+
+    def test_create_zone(self):
+        zone = self.driver.create_zone('example.com')
+        self.assertEquals(zone.domain, 'example.com')
+
+    def test_get_zone(self):
+        zone = self.driver.get_zone('example.com')
+        self.assertEquals(zone.domain, 'example.com')
+        self.assertEquals(zone.id, 'ffb62570-8414-4578-a346-526b44e320b7')
```

Comment:
I am going to merge it, but assertEquals is deprecated so its gives several warnings to use assertEqual instead.
I will let you fix that in the subsequent PR